app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx

```diff
@@ -91,17 +124,19 @@ const inviteFormSelector = formValueSelector('inviteUserForm');
     selectedProject: ownProps.data.isProjectSelector
       ? inviteFormSelector(state, 'project')
       : urlProjectSlugSelector(state),
+    projectName: projectNameSelector(state),
     selectedUser: inviteFormSelector(state, 'user'),
     isAdmin: isAdminSelector(state),
     projectKey: projectKeySelector(state),
     initialValues: {
-      role: DEFAULT_PROJECT_ROLE,
+      role: false,
```

Comment:
role is boolean?

Reply:
roles are determined by the checkbox, false - Viewer, true - Editor

Reply:
Done.